src/NServiceBus.Core/Pipeline/Incoming/MessageProcessingBusConfigurationExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NServiceBus.Pipeline
+{
+    using NServiceBus.Features;
+
+    /// <summary>
+    /// Allows configuring of the message processing feature.
+    /// </summary>
+    public static class MessageProcessingBusConfigurationExtensions
+    {
+        /// <summary>
+        /// Disables the main processing pipeline (by removing all the standard pipeline behaviors). 
+        /// </summary>
+        public static void DisableMainProcessingPipeline(this BusConfiguration busConfiguration)
```

Comment:
SLR gets disabled because it is missing the proper place in the pipeline. Question: should I disable all the features or it is enough to depend on them not working because the behaviors cannot be added to the pipeline?

ErrorQ is enabled because I thought it is generic enough to be useful even here.